src/helpers.ts

```diff
@@ -25,3 +26,25 @@ export function resolveContextValue(context: Context, field: string) {
   }
   return undefined;
 }
+
+export function safeName(str: string = '') {
+  return str.replace(/\//g, '_');
+}
+
+
+export function generateInstanceId(instanceId?: string): string {
+  if(instanceId) {
+    return instanceId;
+  } 
+  let info;
+  try {
+    info = userInfo();
```

Comment:
Can we console.warn this or something? I've spent too much of my life hunting down exceptions that died in an empty catch block.

Reply:
fair point. Its not super important feature, but it can be quite nice way to identify the actual connected instance.

Reply:
after thinking about this I would rather not issue an warning here. This will often just work in a local environment, but still very useful as you will end up seeing the username for that computer in the connected instance id.